Detections/SecurityEvent/UserAccountEnabledDisabled_10m.yaml

```diff
@@ -25,22 +28,57 @@ query: |
   let timeframe = 1d;
   let spanoftime = 10m;
   let threshold = 0;
-  SecurityEvent
+  (union isfuzzy=true
+  (SecurityEvent
   | where TimeGenerated > ago(timeframe+spanoftime)
   // A user account was enabled
   | where EventID == 4722
   | where AccountType =~ "User"
   | where TargetAccount !hassuffix "$"
   | project EnableTime = TimeGenerated, EnableEventID = EventID, EnableActivity = Activity, Computer, UserPrincipalName, 
   AccountUsedToEnable = SubjectAccount, SIDofAccountUsedToEnable = SubjectUserSid, TargetAccount = tolower(TargetAccount), TargetSid
+  ),
+  (
+  WindowsEvent
+  | where TimeGenerated > ago(timeframe+spanoftime)
+  // A user account was enabled
+  | where EventID == 4722 and EventData has "User"
```

Comment:
I think that unless I missed something, the EventData has "User" here is a mistake.